src/Form.php

```diff
@@ -0,0 +1,1202 @@
+			'need_access' => esc_html__('This website is running in a local development environment. To provide support, we must be able to access your site using a publicly-accessible URL.', 'trustedlogin'),
+			'about_live_access_url' => esc_url($this->config->get_setting('vendor/about_live_access_url', self::ABOUT_LIVE_ACCESS_URL)),
+			'learn_more' => esc_html__('Learn more.', 'trustedlogin'),
+		);
+
+		return $this->prepare_output($notice_template, $content);
+	}
+
+	/**
+	 * @return string
+	 */
+	private function get_logo_html()
+	{
+
+		$logo_url = $this->config->get_setting('vendor/logo_url');
+
+		$logo_output = '';
+
+		if (!empty($logo_url)) {
+			$logo_output = sprintf(
+				'<a href="%1$s" title="%2$s" target="_blank" rel="noreferrer noopener"><img src="%3$s" alt="%4$s" /></a>',
+				esc_url($this->config->get_setting('vendor/website')),
+				// translators: %s is replaced with the name of the software developer (e.g. "Acme Widgets")
+				sprintf('Visit the %s website (opens in a new tab)', $this->config->get_setting('vendor/title')),
+				esc_attr($this->config->get_setting('vendor/logo_url')),
+				esc_attr($this->config->get_setting('vendor/title'))
+			);
+		}
+
+		return $logo_output;
+	}
+
+	/**
+	 * Returns the HTML for the footer in the Auth form
+	 *
+	 * @return string
+	 */
+	private function get_footer_html()
+	{
+
+		$support_url = $this->config->get_setting('vendor/support_url');
+
+		if ($reference_id = Client::get_reference_id()) {
+
+			$support_args = array(
+				'tl'  => Client::VERSION,
+				'ref' => $reference_id,
+				'ns'  => $this->config->ns(),
+			);
+
+			$support_url = add_query_arg($support_args, $support_url);
+		}
+
+		$footer_links = array(
+			esc_html__('Learn about TrustedLogin', 'trustedlogin')                   => self::ABOUT_TL_URL,
+			// translators: %s is replaced with the name of the software developer (e.g. "Acme Widgets")
+			sprintf('Visit %s support', $this->config->get_setting('vendor/title')) => $support_url,
+		);
+
+		/**
+		 * Filter trustedlogin/{ns}/template/auth/footer_links
+		 *
+		 * Used to add/remove Footer Links on grantlink page
+		 *
+		 * @since 1.0.0
+		 *
+		 * @param array $footer_links Array of links to show in auth footer (Key is anchor text; Value is URL)
+		 */
+		$footer_links = apply_filters('trustedlogin/' . $this->config->ns() . '/template/auth/footer_links', $footer_links);
+
+		$footer_links_output = '';
+		foreach ($footer_links as $text => $link) {
+			$footer_links_output .= sprintf(
+				'<li><a href="%1$s" target="_blank">%2$s</a></li>',
+				esc_url($link),
+				esc_html($text)
+			);
+		}
+
+		$footer_output = '';
+		if (!empty($footer_links_output)) {
+			$footer_output = sprintf('<ul>%1$s</ul>', $footer_links_output);
+		}
+
+		return $footer_output;
+	}
+
+	private function prepare_output($template, $content, $wp_kses = true)
```

Comment:
I feel like this should be a static method in its own `Helpers.php` file or something like that.

Reply:
It's only need  in this class though